bandit/reporters/xunit.h

```diff
@@ -101,16 +94,15 @@ namespace bandit {
       }
 
       void print_remaining_header_with_time() {
-        auto dur = std::chrono::high_resolution_clock::now() - testcase_start_time_point_;
-        testsuite_runtime_ += std::chrono::duration_cast<std::chrono::nanoseconds>(dur);
-        std::chrono::duration<double> dur_in_sec(dur);
-        work_stm_ << " time=\"" << std::to_string(dur_in_sec.count()) << "\">\n";
+        if (report_timing_) {
+          std::chrono::duration<double> dur_in_sec(testcase_duration_);
+          work_stm_ << " time=\"" << std::to_string(dur_in_sec.count());
+        }
+        work_stm_ << "\">\n";
```

Comment:
According to the XML specification(*), `time` is a required attribute, so I'd prefer if it says `time="0.0"` if the feature is disabled. This is also the behavior of former bandit versions.

(*) I am ignoring the fact that we don't comply to the specification regarding other things, but... well :sweat_smile:

Reply:
Fixed here: 47232d5

What about displaying a warning a warning on `std::clog` or `std::cerr` if reporter `xunit` is specified without `--report-timing`?

Reply:
Other question: if we display always the `time` attribute for `xunit` reporter, should I use [std::thread::sleep_for](http://www.cplusplus.com/reference/thread/this_thread/sleep_for/) to avoid zero value?

Reply:
I have also thought about that but I think no, we just assume zero time in the tests, although this could mean that tests fail under big load.